template/en/default/admin/components/edit-common.html.tmpl

```diff
@@ -94,6 +94,11 @@
        %]
     </td>
 </tr>
+<tr>
+  <th class="field_label"><label for="team_name">Team Name:</label></th>
+  <td><input size="64" maxlength="64" name="team_name" id="team_name"
+             value="[%- comp.team_name || 'Mozilla' FILTER html %]"></td>
```

Comment:
Please make this an autocomplete field pulling from the list of current team names (we already use `devbridgeAutocomplete` in a few places for this).

That'll make it easier to avoid typos when using the same name in multiple components.

Reply:
As you noted this still needs autocomplete.

Reply:
Might be better to default to an empty value to force thought around selecting the correct team name.